core/src/main/java/io/grpc/NameResolver.java

```diff
@@ -272,6 +272,24 @@ public SynchronizationContext getSynchronizationContext() {
      */
     public static final class ConfigOrError<T> {
 
+      private static final class Unsuppported {
+
+        Unsuppported() {}
+
+        @Override
+        public String toString() {
+          return "service config parsing not supported";
+        }
+      }
+
+      /**
+       * A sentinel value indicating that service config is not supported.   This can be used to
+       * indicate that parsing of the service config is neither right nor wrong, but doesn't have
+       * any meaning.
+       */
+      public static final ConfigOrError<?> UNSUPPORTED_CONFIG =
```

Comment:
I don't think "unsupported" is accurate here. Service config is not something a LB policy **supports** or not, it's a mechanism that the policy **uses** or not. If `DUMMY_CONFIG` doesn't sound good to you, maybe `NO_CONFIG` or `UNUSED_CONFIG`?